lib/parser/provenance.h

```diff
@@ -208,6 +218,9 @@ class CookedSource {
   bool IsValid(ProvenanceRange r) const { return allSources_.IsValid(r); }
 
   std::optional<ProvenanceRange> GetProvenanceRange(CharBlock) const;
+  std::optional<CharBlock> GetCharBlockFromLineAndColumns(int, int, int) const;
```

Comment:
Here's a case where argument names would reduce the mystery of the three `int` arguments.